master/buildbot/status/web/forms.py

```diff
@@ -34,9 +35,56 @@ def content(self, request, cxt):
     def getChild(self, path, req):
         if path == "forceBuild":
             return ForceBuildDialogPage()
+        elif path == "rebuild":
+            return RebuildDialogPage()
 
+class BuildDialogPage(HtmlResource):
+    def _decodeArgs(self, request, encoding):
+        args = request.args.copy()
+        for name, argl in args.iteritems():
+            if '_branch' in name:
+                args[name] = [ self.decodeFromURL(arg, encoding) for arg in argl ]
```

Comment:
[minor] in pep8 some designed for python style guides shouldn't be there any spaces next to `[]`